### PR TITLE
Highlight thumbnails when clicking next/prev buttons

### PR DESCRIPTION
**Pull request by BC-Mikey**
FORD-73

### CODE REVIEW

File: lib/bcgs_ford_filmstrip.js [hunk 1]
```diff
@@ -134,10 +134,12 @@ v1.0.0: 6/19/2015
        
         onNextVideo = function(event) {
             bcc.trigger(bcc.events.REQUEST_NEXT_VIDEO);
+            carousel.highlightIndex();
         },
        
         onPrevVideo = function(event) {
             bcc.trigger(bcc.events.REQUEST_PREVIOUS_VIDEO);
+            carousel.highlightIndex();
         },
```
Comment (bceacuna) [inline lib/bcgs_ford_filmstrip.js]: I was expecting to see highlightIndex() definitaion in this request. If highlightIndex() is already in place, merge away

Comment (BC-Mikey) [inline lib/bcgs_ford_filmstrip.js]: What do you mean? I added it to both onPrevVideo() and onNextVideo()

Comment (bceacuna) [inline lib/bcgs_ford_filmstrip.js]: My question is more around, Where is the highlightIndex method defined?

Comment (BC-Mikey) [inline lib/bcgs_ford_filmstrip.js]: It's a method of the carousel class, so its part of bcc_carousel.js

**Comment by bceacuna** [comment]
Looks good. Merge away.

**Pull request merged**